tokio/src/lib.rs

```diff
@@ -52,6 +52,13 @@
 //! control what features are present, so that unused code can be eliminated.
 //! This documentation also lists what feature flags are necessary to enable each API.
 //!
+//! The easiest way to get started is to enable all features. Do this by
```

Comment:
:thumbsup:

can we put this in the README as well?

Reply:
updated